pycombo/misc.py

```diff
@@ -80,7 +80,7 @@ def deconstruct_graph(graph, weight: Optional[str] = None) -> Tuple[nodes_, edge
         if is_weighted(graph, weight=weight):
             default_ = 0
         else:
-            logger.info(f"No property found: `{weight}`. Using as unweighted graph")
```

Comment:
Why did you make than change? Personally I'd prefer backtick as in markdown it highlights code/numeric values

Reply:
Flake8 4.0.1 on Python 3.12 complained about these backticks, so I changed them. Apparently, that was a bug, which was fixed in newer versions of flake8. I reverted this change.